skins/wmde19/.browserslistrc

```diff
@@ -0,0 +1,3 @@
+> 1%
+last 2 versions
+not ie <= 8
```

Comment:
As per the [Mediawiki Browser Support Matrix](https://www.mediawiki.org/wiki/Compatibility#Browser_support_matrix), we only support IE 11. Yay!